src/Mpdf.php

```diff
@@ -12964,6 +12962,10 @@ function WriteHTML($html, $sub = 0, $init = true, $close = true)
 			}
 		}
 
+		if (in_array($mode, HTMLParserMode::getAllModes(), true) === false) {
+			throw new \Mpdf\MpdfException('WriteHTML() requires $mode to be one of the modes defined in HTMLParserMode');
```

Comment:
Can we, for the time being, use a less-strict warning here? Eg throwing an exception only when the debug mode is on? This is IMHO somewhat a BC break, so I'd be reluctant to incorporate this in a minor release.

Reply:
It might also be a good idea to default `$mode` to `HTMLParserMode::DEFAULT_MODE` if the user passes in an unknown mode for backwards compatibility reasons.